attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/AttributeServiceImpl.java

```diff
@@ -180,16 +180,18 @@ public void updateSourceMetadata(
   }
 
   @Override
-  public void delete(AttributeMetadataFilter request, StreamObserver<Empty> responseObserver) {
-    validateDeletionFilter(request);
+  public void delete(
+      final AttributeMetadataFilter request, final StreamObserver<Empty> responseObserver) {
+    final AttributeMetadataFilter modifiedRequest = validateAndUpdateDeletionFilter(request);
```

Comment:
Had a similar comment on one of the other PRs, but if this throws, nothing is catching it and reporting it to the osberver.

Reply:
Good catch. Handled now.